prototyping/mocking/mockSearch.cpp

```diff
@@ -0,0 +1,48 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+void search(unordered_set<string> words, vector<string> articles) {
```

Comment:
I think defining the return here/printing it out is important for prototyping purposes. I see the ranking algorithm for what order you return articles in, and how you use NLP to help improve that, as the core functionality of the query engine.

Reply:
Here I am just counting incidences. The nlp is sort of preprocessing for this algorithm. Full explanation is on my individual design doc.